salt/cloud/clouds/opennebula.py

```diff
@@ -1053,7 +1053,13 @@ def __query_node_data(vm_name):
         try:
             private_ip = data['private_ips'][0]
         except KeyError:
-            private_ip = data['template']['nic']['ip']
+            try:
+                private_ip = data['template']['nic']['ip']
+            except:
```

Comment:
This needs to be `except Exception:` because we don't allow bare excepts. Thanks!